### ISSUE TITLE
Code in Chapter 6 Exercises for 'def InterpolateSample' thows an error

### ISSUE THREAD

**Issue by resonates7** [AllenDowney/ThinkStats2#179]
'def InterpolateSample' throws an error b/c NumPy cannot convert a float64 type to an integer inside that function. The following line of code needs to be changed to convert the 'freq' value to an integer. Easy fix, but took me a minute to figure it out.

        vals = np.linspace(row.log_lower, row.log_upper, row.freq.**astype(np.int64)**)

**Comment by thakkery** [comment]
I had the same issue. I did:

```python
vals = np.linspace(row.log_lower, row.log_upper, int(row.freq))
```

**Comment by AllenDowney** [comment]
This should be resolved by e136acc1047f4dbc3cd9bc7935420b8d6f92ce40
